docs/streams/developer-guide/config-streams.html

```diff
@@ -310,6 +312,40 @@ <h4><a class="toc-backref" href="#id5">bootstrap.servers</a><a class="headerlink
                 <li><a class="reference external" href="../../../javadoc/org/apache/kafka/streams/errors/LogAndFailExceptionHandler.html">LogAndFailExceptionHandler</a>.
                   This handler logs the deserialization exception and then signals the processing pipeline to stop processing more records.</li>
               </ul>
+
+              <p>You can also provide your own customized exception handler besides the library provided ones to meet your needs. For example, you can choose to forward corrupt
+                records into a quarantine topic (think: a “dead letter queue”) for further processing. To do this, use the Producer API to write a corrupted record directly to
+                the quarantine topic. The drawback of this approach is that “manual” writes are side effects that are invisible to the Kafka Streams runtime library,
+                so they do not benefit from the end-to-end processing guarantees of the Streams API:</p>
+
+              <pre class="brush: java;">
+              class SendToDeadLetterQueueExceptionHandler implements DeserializationExceptionHandler {
```

Comment:
I think we should not indent here, because this would be rendered a look strange.

Reply:
Should it be `public class` ?

Reply:
The indent would actually be ignored in `pre` I think: https://kafka.apache.org/11/documentation/streams/developer-guide/config-streams.html#default-production-exception-handler

Reply:
Ack, also adding `public` to `IgnoreRecordTooLargeHandler`.